packages/react/src/themes/teams/components/Menu/menuVariables.ts

```diff
@@ -12,7 +12,6 @@ export const menuColorAreas = stringLiteralsArray(
   'backgroundHover',
   'backgroundActive',
   'backgroundFocus',
-  'foregroundFocus',
```

Comment:
This is used in the `menuItemStyles.ts` why is it removed? Btw we should have typescript error in the menuItemStyles.ts after this change

Reply:
This one was in the list twice (line 11)